viewing_party/main.py

```diff
@@ -0,0 +1,136 @@
+def create_movie(title, genre, rating):
+    movies_dict = {}
+    if title and genre and rating:
+        movies_dict["title"] = title
+        movies_dict["genre"] = genre
+        movies_dict["rating"] = rating
+        return movies_dict
+    else:
+        return None
+
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title in movie["title"]:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            return user_data
+    else:
+        return user_data
+
+
+##### WAVE 2 #####
+
+def get_watched_avg_rating(user_data):
+    ratings_from_user = []
+    if len(user_data["watched"]) == 0:
+        return 0.0
+    else:
+        for movies in user_data["watched"]:
+            ratings_from_user.append(movies["rating"])
+        total_sum = sum(ratings_from_user)
+        avg_rating = total_sum/(len(ratings_from_user))
+        return avg_rating
+
+
+def get_most_watched_genre(user_data):
+    genres_from_user = []
+    if len(user_data["watched"]) == 0:
+        return None
+    else:
+        for movies in user_data["watched"]:
+            genres_from_user.append(movies["genre"])
+        most_popular_genre = max(
```

Comment:
Nice use of max and set here. Please add a comment next time you use code that's outside of the scope of the curriculum.